autogpt/agent/agent.py

```diff
@@ -166,7 +166,11 @@ def signal_handler(signum, frame):
                         assistant_reply_json, assistant_reply, self.config
                     )
                     if self.config.speak_mode:
-                        say_text(f"I want to execute {command_name}")
+                        command_vars = vars(
+                            self.command_registry.commands[command_name]
+                        )
+                        description = command_vars["description"]
+                        say_text(command_name, description)
```

Comment:
please use copy command to make sure you don't overwrite the pointer.
It is unlikely to overwrite but still happened in the memory functions for example.
copy.copy(description) should be fine.

Reply:
What pointer do you mean? Strings are passed by value in Python.